### PR TITLE
RHMAP-18741 Make Digger Docs navigation better

### PR DESCRIPTION
**Pull request by finp**
Make it easier to navigate to digger docs:
* add it as a top bar navigation item (replacing Road Map, which has not been updated for years)
* repurpose digger-install-intro.asciidoc as index.markdown to provide landing page with navigation to all guides

### CODE REVIEW

**Comment by laurafitzgerald** [comment]
@finp I think that the 'get started` link/button here: '<url>'/digger/ is also misleading. It leads the user to information about clients which is useful for those wishing to contribute. I think we should somehow differentiate between pages that are useful for possible contributors and a user who wants to install and use the tool. wdyt?

File: docs/digger/index.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+* A Jenkins server that is used to run the build tasks
+* A Docker-formated image with all the required tools to build Android applications
+* macOS servers with pre-installed tools to build iOS applications (optional)
+* An http://docs.ansible.com/ansible/index.html[Ansible] based automation tool used to install and configure all the components
+* A nagios server for monitoring the digger components
+* A suite of sample Jenkinsfiles to demonstrate how to build mobile applications using Jenkins pipeline
+* [Client applications](/digger) to help you use Digger.
+
+This project is designed to be used with the RedHat Mobile Application Platform (RHMAP).
```
Comment (laurafitzgerald) [inline docs/digger/index.markdown]: We should remove this reference to RHMAP

**Comment by laurafitzgerald** [comment]
also I think the learn about digger link on the homepage should land here `<url>`/docs/digger/

File: docs/digger/index.markdown [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+layout: index
+title: Digger
+section: digger
+---
+
+<h2 class="section-header" id="digger"><img src="/img/aerogeardigger_icon_32px_cropped.png"> AeroGear<strong>Digger</strong> Guides</h2>
+___
+<p>
+<a href="https://github.com/aerogear/digger-installer" class="btn btn-primary-inverse"><i class="fa fa-cloud-download"></i> AeroGear Digger Installer</a>
```
Comment (laurafitzgerald) [inline docs/digger/index.markdown]: The repo name has been changed to `https://github.com/aerogear/aerogear-digger-installer` and could we also make this open a new tab

File: docs/digger/index.markdown (hunk 1 above)
Comment (matzew) [reply docs/digger/index.markdown]: I'd think that it's worth to mention that RHMAP does integrate it.

But it's designed to run independently - Digger should not know anything about RHMAP

**Comment by finp** [comment]
closing in favour of a more comprehensive solution

**Pull request closed without merging**